app/sql.js

```diff
@@ -837,11 +852,27 @@ async function updateToLokiSchemaVersion1(currentVersion, instance) {
   const publicChatData = {
     ...baseData,
     id: 'publicChat:1@chat.lokinet.org',
-    server: 'https://chat.lokinet.org',
+    server: 'chat.lokinet.org',
```

Comment:
Can we use `lokiPublicServerData.server` here or are these two values actually independent but happen to be equal?

Reply:
Yes we could good call, this is essentially a foreign key